manifests/o/OBSProject/OBSStudio/Beta/28.0.0-beta1/OBSProject.OBSStudio.Beta.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: OBSProject.OBSStudio.Beta

```

```suggestion
PackageIdentifier: OBSProject.OBSStudio.Pre-release
```
Maybe this should be moved to `OBSProject.OBSStudio.Pre-release` since that includes all pre-release (beta) builds.